resources/dashboards/Ingress-Egress-Traffic-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "hits": 0, 
+   "timeRestore": false, 
+   "description": "", 
+   "title": "Ingress Egress Traffic Dashboard", 
```

Comment:
@craig-cogdill I have not discussed it with Rob and I don't know if the team has discussed it either but since there is a **strong** correlation (1:1) between the DPA rule and this dashboard I think the naming is key. 

Unless you have already come up with a naming policy for DPA Rules --> Dashboards then we need to have that talk in the team 

*DPA-Ingress-Egress-Traffic-Dashboard.json*?    ... Also, depending on the kibana cache issue (see comment below) this probably warrants some extra attention in the docs which needs to be communicated to JD.